misk/build.gradle

```diff
@@ -71,3 +71,5 @@ afterEvaluate { project ->
     outputFormat = 'gfm'
   }
 }
+
+apply from: "$rootDir/gradle-mvn-publish.gradle"
```

Comment:
would it be simpler to add this to the allprojects clause in the root project? (And probably remove the same for GFM config)